EveripediaNetwork/iq-ui
# Fixes #1203: NFT Asset addition to the Treasury page

## NFT Asset addition to the Treasury page 

This pull request fixes the bond link in the treasury section and adds a new NFT to the treasury list.

Details:

 1. Fixed bond link: The bond link in the sidebar was broken and redirected to an error page. I have fixed it by updating the link to the correct URL.

 2. Added NFT to treasury list: I have also added a new NFT to the treasury list.

Overall, this pull request enhances the user experience by ensuring that the bond link is functional and that our treasury list is up-to-date with the latest additions.

## How should this be tested?

1. Check the treasury page by clicking treasury on the sidebar panel, scrolling down then you see the new addition.
2. Click bonds on the sidebar panel and it will redirect you to the new page
